docs/reference/UDS Core/IdAM/upgrading-versions.md

```diff
@@ -51,7 +57,7 @@ In uds-identity-config versions v0.11.0+, the UDS Operator can automatically swi
                   - Description `UDS Client Credentials`
                   - Click `Duplicate`
          - Go to `Authentication` > `UDS Client Credentials`
-               - Click `Add Step`
+               - Click `Add Execution`
```

Comment:
On my deployed version (uds-core 0.39.0) this WAS `Add Step`. Not sure if it varies across Keycloak versions though.

Reply:
Yes! The Keycloak stock theme changed over the versions 🤦🏻